apps/marketplace/src/app/faq/faq.component.html

```diff
@@ -0,0 +1,19 @@
+<h1>Frequently Asked Questions</h1>
+
+<h2>How to set up MetaMask</h2>
+<p>
+  Follow this article on Medium step by step and you'll be good to go in no-time!
+</p>
+<mat-card (click)="navigateTo('https://medium.com/@mail.bahurudeen/setup-a-metamask-ethereum-wallet-and-use-it-to-send-and-receive-ether-4f3b99360e4f')">
+  <mat-card-header>
+    <h3>Setup a MetaMask Ethereum Wallet and use it to Send and Receive Ether.</h3>
+  </mat-card-header>
+  <mat-card-content fxLayout fxLayoutGap="12px">
+    <section fxLayout="column">
+      Let’s see what MetaMask is, how to setup and use it with Rinkeby Test Network.
+    </section>
+    <section fxLayout="column">
+      <img [src]="'https://miro.medium.com/max/3028/1*4oX18KhwPPzXuyaPnOIHyw.png'">
+    </section>
+  </mat-card-content>
+</mat-card>
```

Comment:
I'm lazy so I just link to an article that explains how to setup MetaMask and add Rinkeby tokens